.github/workflows/ci.yaml

```diff
@@ -1844,6 +1835,36 @@ jobs:
           ignore-unfixed: true
           vuln-type: 'os,library'
           severity: 'CRITICAL,HIGH'
+  scan-cactus-connector-quorum:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3.5.2
+      - name: Build an image from Dockerfile
+        run: DOCKER_BUILDKIT=1 docker build ./packages/cactus-plugin-ledger-connector-quorum/ -f ./packages/cactus-plugin-ledger-connector-quorum/Dockerfile -t cactus-connector-quorum
+      - name: Run Trivy vulnerability scan for cactus-plugin-ledger-connector-quorum
+        uses: aquasecurity/trivy-action@0.11.2
+        with:
+          image-ref: 'cactus-connector-quorum'
+          format: 'table'
+          exit-code: '1'
+          ignore-unfixed: true
```

Comment:
Why are we setting this field to `true`?

Reply:
Addressed and set this to `false`